common/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "circuit-common"
+name = "common"
```

```suggestion
name = "zk-circuits-common"
```

We should probably be a little more specific